src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
```

Comment:
The `alt` attribute should describe the image content. Use a more descriptive alternative text for the logo image, such as 'Moyo company logo'.